src/raftstore/errors.rs

```diff
@@ -55,15 +55,15 @@ quick_error!{
             display("{:?}", err)
         }
 
-        // Following is for From other errors.
+// Following is for From other errors.
```

Comment:
This is the behaviour of rustfmt, looks like different version of rustfmt.
I will fix it.